openshift_tools/ircbot/openshift_sre/openshift_sre.py

```diff
@@ -0,0 +1,351 @@
+# coding=utf-8
+"""
+openshift_sre.py - Various helper modules for OpenShift Online SRE team
+Copyright © 2017, Will Gordon <wgordon@redhat.com>
+Licensed under the MIT license.
+"""
+import re
+from datetime import datetime as dt
+from sopel import module
```

Comment:
sopel is not in the jenkins test machine. You'll need to ignore this error with:

```
from sopel import module # pylint: disable=import-error
```